modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_folder.jsp

```diff
@@ -215,11 +215,22 @@ renderResponse.setTitle(dlEditFolderDisplayContext.getHeaderTitle());
 
 					<c:if test="<%= dlEditFolderDisplayContext.isSupportsPermissions() %>">
 						<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" label="permissions">
-							<liferay-ui:input-permissions
-								modelName="<%= DLFolderConstants.getClassName() %>"
-								permissionPropagationCheckboxLabel="set-permissions-for-documents-and-folders-uploaded-to-this-folder"
-								permissionPropagationCheckboxValue="<%= true %>"
-							/>
+							<c:choose>
+								<c:when test="<%= dlEditFolderDisplayContext.getInheritableParentFolderId() >= 0 %>">
+									<label class="font-weight-normal mb-3">
+										<liferay-ui:message arguments='<%= "<strong>" + HtmlUtil.escape(dlEditFolderDisplayContext.getInheritableParentFolderName()) + "</strong>" %>' key="these-permissions-are-inherited-from-the-parent-folder" />
```

Comment:
Not sure. However, I think it is better to change  -> getInheritedParentFolderName

Reply:
`Inheritable` means `capable of being inherited` and I think it is more correct in this case than `Inherited`. Thanks.